src/components/registration/registration.html

```diff
@@ -0,0 +1,49 @@
+<!-- Generated template for the RegistrationComponent component -->
+<ion-content padding class="registration">
```

Comment:
components dont have their own ion-content, they live inside the ion-content on the page html. Please fix